layouts/partials/version-banner.html

```diff
@@ -1,13 +1,14 @@
-<!-- Match the page URL (permalink) against the regex pattern for archived doc
-  sets. If the pattern matches, display a banner. -->
-{{ $ismatch := findRE .Site.Params.archive_url_pattern .Permalink }}
-{{ if $ismatch }}
+<!-- Check the variable that indicates whether this is an archived doc set.
+  If yes, display a banner. -->
+{{ if .Site.Params.archived_version }}
   {{ $color := "primary" }}
+  {{ $latest := .Site.Params.url_latest_version }}
   <div class="pageinfo pageinfo-{{ $color }}">
     {{ with .Site.Params.version }}<p>Version {{ . | markdownify }} of the
       documentation is no longer actively maintained. The site that you are
       currently viewing is an archived snapshot. For up-to-date documentation,
-      see the <a href="https://kubeflow.org/docs/">latest version</a>.</p>
+      see the 
+      <a href="{{ $latest | safeURL }}">latest version</a>.</p>
```

Comment:
hello @sarahmaddox , I read through the updates. I like the single parameter setting to trigger the include of the banner on the page. Does this partial place a banner on the archived pages or is there a different issue?

Reply:
Hallo @kbhawkey Thanks for the review and feedback! This partial places the banner, but this PR applies only to the master branch, which affects only the most recent version of the docs.

As a next step, I'll need to apply the same changes to all the archive branches (`v0.6-branch`, `v0.5-branch`, and so on). I'll need to update the parameter on those branches, so that the branches display the banner. Those will be separate PRs. 

In addition, I'd like to get this PR merged into master before we create the `v0.7-branch` from master, so that I can test the whole versioning procedure with a branch that already contains the partial.

Reply:
:+1: